src/Common/perf/PerfRunner/PerfRunner.cs

```diff
@@ -0,0 +1,30 @@
+using System.IO;
+using System.Reflection;
+using System.Collections.Generic;
+using Microsoft.Xunit.Performance.Api;
+
+public class PerfHarness
+{
+    public static void Main(string[] args)
+    {
+        using (XunitPerformanceHarness harness = new XunitPerformanceHarness(args))
```

Comment:
This create only one ETL file perf assembly that runs.  That ETL can some times be quite large (4GB) because we have enough tests in one assembly that running the tests taks about 15 minutes.  But we do not just have one large ETL file for all the assemblies.